0002-quilt.mod.json.md

```diff
@@ -0,0 +1,254 @@
+
+If an entrypoint does not need to specify a language adapter other than the default language adapter, the entrypoint can bre represented simply as the value string instead.
+
+
+### The `plugins` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of loader plugins. A plugin is an object with the following keys:
+* adapter — Language adapter to use for this plugin
+* value — Points to an implementation of the `LoaderPlugin` interface. Can be in either of the following forms:
+    * `my.package.MyClass` — A class to be instantiated and used
+    * `my.package.MyClass::thing` — A static field containing an instance of a `LoaderPlugin`
+
+If a plugin does not need to specify a language adapter other than the default language adapter, the plugin can bre represented simply as the value string instead.
+
+### The `jars` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+A list of paths to nested JAR files to load, relative to the root directory inside of the mods JAR.
+
+### The `languageAdapters` field
+| Type   | Required |
+|--------|----------|
+| Object | False    |
+
+A collection of `"key": value` pairs, where each key is the namespace of a language adapter and the value is an implementation of the `LanguageAdapter` interface.
+
+### The `mixins` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of paths to mixin configuration files relative to the root of the mod JAR.
+
+### The `accessWidener` field
+| Type   | Required |
+|--------|----------|
+| Array  | False    |
+
+An array of paths to accesswidener files relative to the root of the mod JAR.
+
+### The `depends` field
+| Type   | Required |
+|--------|----------|
+| Object | False    |
+
+Defines mods that this mod will not function without.
+
+A collection of `"key": value` pairs, where each key is in the form of either `mavenGroup:modId` or `modId` and each value is either a version range specifier or array of version range specifiers. If an array of range specifiers is provided, the version matches if it matches ANY of the listed specifiers. A version range specifier can make use of any of the following patterns:
+* `*` — Matches any version. Will fetch the latest version available if needed
+* `=1.0.0` — Matches exactly version 1.0.0 and no other versions
+* `>=1.0.0` — Matches any version greater than or equal to 1.0.0
+* `>1.0.0` — Matches any version greater than version 1.0.0
+* `<=1.0.0` — Matches any version less than or equal to version 1.0.0
+* `<1.0.0` — Matches any version less than version 1.0.0
+* `1.0.x` — Matches any version with major version 1 and minor version 0.
+* `~1.0.0` — Matches the most recent version greater than or equal to version 1.0.0 and less than 1.1.0
+* `^1.0.0` — Matches the most recent version greater than or equal to version 1.0.0 and less than 2.0.0
```

Comment:
Rust treats unqualified as `^` because patch bumps aren't supposed to have major breaking changes. I guess my only concern would be Minecraft itself, ex. 1.16.2